Source/Mock.cs

```diff
@@ -667,6 +669,44 @@ private static Expression GetValueExpression(object value, Type type)
 			return Expression.Convert(Expression.Constant(value), type);
 		}
 
+		[SuppressMessage("Microsoft.Design", "CA1006:DoNotNestGenericTypesInMemberSignatures", Justification = "By Design")]
+		internal static SetupSequencePhrase<TResult> SetupSequence<T, TResult>(Mock<T> mock, Expression<Func<T, TResult>> expression)
+			where T : class
```

Comment:
Does this method really have to be generic? Why not make the parameters be of types `Mock` and `LambdaExpression`? The public method in `Mock<T>` can make sure that everything is appropriately typed.